sys/riotboot/Makefile.include

```diff
@@ -1,7 +1,15 @@
-# Indicate the reserved space for a header, 256B by default
-# Notice that it must be 256B aligned. This is restricted by
-# the Cortex-M0+/3/4/7 architecture
-RIOTBOOT_HDR_LEN ?= 0x100
+# Indicate the reserved space for a header.
+# Notice that it must be at least 256B aligned. This is restricted by
+# the Cortex-M0+/3/4/7 architecture. "The Vector table must be naturally
+# aligned to a power of two whose alignment value is greater than or equal
+# to number of Exceptions supported x 4"
```

Comment:
It would be good if you have a link for this information, it would be useful for porting future boards and justifying the values.